gateway/src/http_entrypoints.rs

```diff
@@ -625,6 +674,28 @@ async fn sp_startup_options_set(
     Ok(HttpResponseUpdatedNoContent {})
 }
 
+/// Read the current value of a sensor by ID
+///
+/// Sensor IDs come from the host topo tree.
+#[endpoint {
+    method = GET,
+    path = "/sp/{type}/{slot}/sensor/value/{sensor_id}",
```

Comment:
Slight API nit, but I kind of expected the `{sensor_id}` and `value` path components to be flipped. I think, anyway? Actually, is there anything to `GET` on the sensor _other_  than the value? Could we get rid of the `value` path component entirely?

Reply:
They definitely should be flipped - not sure what I was thinking there.

There's nothing to `GET` on the sensor today other than the value, but I wondered if we might add something like "get the description" or "get the units" in the future. I could see an argument for making this just `.../sensor/{sensor_id}`, and if we add metadata like that in the future, it could hang off of `.../sensor/{sensor_id/description` or whatever?

Reply:
Yeah, if you'd like to leave it open to get future data under the sensor path, then  `.../sensor/{sensor_id}/value` makes sense!